journal/journal_unix.go

```diff
@@ -126,6 +121,16 @@ func Send(message string, priority Priority, vars map[string]string) error {
 	return nil
 }
 
+// getConn attempts to get the global `unixConnPtr` socket, initializing if necessary
+func getConn() *net.UnixConn {
```

Comment:
I guess `getOrInitConn` may be a better name, perhaps?

The docstring on `initConn` should also be updated to reflect the new situation.